Package.swift

```diff
@@ -0,0 +1,62 @@
+// swift-tools-version:5.0
```

Comment:
Can we omit this? We do not depend on any Swift version, so no need to limit it

Reply:
As per [documentation](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#about-the-swift-tools-version) that line must be first. Not sure if we should omit it